### ISSUE TITLE
Unclear difference between "browse the directory structure" and "open folders"

### ISSUE THREAD

**Issue by GenerAhl** [MicrosoftDocs/windows-itpro-docs#9461]
For the section "If you enable this setting, users can browse the directory structure of the selected drives in My Computer or File Explorer, but they cannot open folders and access the contents. Also, they cannot use the Run dialog box or the Map Network Drive dialog box to view the directories on these drives." what is the difference between "users can browse the directory structure" and "they cannot open folders"? To me, browse a directory structure includes navigating through the folders.

Should it be "but they cannot open folders to access the contents" ?

Could this be clarified further?
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6fe02a28-215f-f176-6086-8e79fcb48471
* Version Independent ID: f9143995-669d-be4d-3632-be2d0517d452
* Content: [Policy CSP - ADMX_WindowsExplorer - Windows Client Management](https://docs.microsoft.com/en-us/windows/client-management/mdm/policy-csp-admx-windowsexplorer#admx-windowsexplorer-noviewondrive)
* Content Source: [windows/client-management/mdm/policy-csp-admx-windowsexplorer.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/client-management/mdm/policy-csp-admx-windowsexplorer.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @ManikaDhiman
* Microsoft Alias: **dansimp**

**Comment by illfated** [comment]
Fair question. Looks like a good suggestion.

Maybe we just need to verify how it needs to be written to adhere to Microsoft Docs conventions, and to clarify the actual difference between
- Browse the directory structure (setting Enabled)
- Open folders and access the contents (setting Disabled)

Looking forward to suggestions from the tech writers to how this can be clarified properly.

**Comment by joinimran** [comment]
Hello @GenerAhl - Thank you for your query. As per descriptions of the content, it says that users can browse the directory structure of the selected drive, this means that you can browse the directories like (see image below).
![image](https://user-images.githubusercontent.com/47118050/119478308-254dbf00-bd69-11eb-88e0-7722bbbe70cc.png)

As you can see that we can browse through the directory but can't access the contents of the folder. Like we can do in explorer as 
![image](https://user-images.githubusercontent.com/47118050/119478615-61811f80-bd69-11eb-95fb-521bc2e3fbc9.png)

Hope this clarifies everything.

Thanks
Imran.

**Comment by GenerAhl** [comment]
@joinimran thanks for your explanation but unfortunately it only states the same thing as I was asking :)
To me, opening a folder would reveal any (potential) folders in it and opening folders within that folder would reveal folders there, essentially "browse the directory structure".
Hence, my point is that we should not state that the user cannot open folders (if they cannot open folders they cannot browse the folder structure) but instead state that they cannot "open files" or potentially even "see the files".

In short: I don't like the wording "open folders" as we **do** allow users to open the folders, we simply only show folders in that folder, not any files.

Is this making more sense?

**Comment by illfated** [comment]
> In short: I don't like the wording "open folders" as we **do** allow users to open the folders, we simply only show folders in that folder, not any files.

Thanks. To me, it makes sense, because I was trying to find a way to describe the fact that we are able to traverse the folder structure and se subfolders, hence we are able to browse and open folders, but not see the file content or data content (as presented by file type or icons).

We should find a Microsoft approved way to state that it is possible to browse the directory structure, see subfolders, but not access any file data in the folders.

**Comment by nam31** [comment]
Closing via #9574 
Thank you
